src/successModal/index.tsx

```diff
@@ -1,100 +1,84 @@
+
+  .kirk-modal-dialog {
+    display: flex;
+    justify-content: center;
+    padding: 0;
     margin: 0 auto;
-    max-width: ${modalSize.m};
-    height: 100%;
-    padding: ${space.xl} 0 ${footerHeight} 0;
+    width: 100%;
+    border-radius: 0;
     background-color: ${color.successBackground};
   }
 
-  /* Stronger selector to override Modal component styling */
-  &.kirk-modal-dimmer .kirk-modal.kirk-successModal .kirk-modal-dialog.kirk-modal-dialog {
-    width: auto;
-  }
-
-  & .kirk-modal--large.kirk-successModal .kirk-modal-dialog {
-    padding: ${space.xl} 0;
-  }
-
-  & .kirk-modal.kirk-successModal .kirk-modal-body {
-    padding: ${space.xl};
-    font-size: ${font.l.size};
-    line-height: ${font.l.lineHeight};
-    text-align: center;
-    align-items: center;
-  }
-
-  & .kirk-modal--large.kirk-successModal .kirk-modal-body {
-    position: relative;
+  .kirk-modal-body {
     display: flex;
-  }
-
-  & .kirk-modal--large .kirk-successModal-bodyItem {
-    width: 50%;
-  }
+    min-height: 100vh;
+    flex-direction: column;
 
-  & .kirk-modal--large .kirk-successModal-bodyItem:nth-child(odd) {
-    padding-right: ${space.xl};
-  }
-
-  & .kirk-modal--large .kirk-successModal-bodyItem:nth-child(even) {
-    padding-left: ${space.xl};
-  }
-
-  & .kirk-successModal-image {
-    width: 80%;
-    margin-bottom: ${space.xl};
+    @media (${responsiveBreakpoints.isMediaLarge}) {
+      flex-direction: row;
+      align-items: center;
+      flex-wrap: wrap;
+    }
   }
+`
 
-  & .kirk-modal--small .kirk-successModal-image {
-    max-height: 33vh;
-    object-fit: contain;
-  }
+const StyledIllustratedSection = styled(IllustratedSection)`
+  display: flex;
+  flex: 1;
 
-  & .kirk-modal--large .kirk-successModal-image {
-    margin-bottom: 0;
-  }
+  .kirk-illustratedSection-illustration {
+    padding-top: calc(2 * ${space.xxl});
+    justify-content: center;
+    align-items: center;
 
-  & .kirk-successModal-footer {
-    position: absolute;
-    bottom: 0;
-    left: 0;
-    width: 100%;
-    height: ${footerHeight};
-    padding: ${space.xl};
-    text-align: center;
+    @media (${responsiveBreakpoints.isMediaLarge}) {
+      padding-top: 0;
+    }
   }
 
-  & .kirk-modal--large .kirk-successModal-footer {
-    position: relative;
-    display: block;
-    margin: 0 auto;
+  .kirk-illustratedSection-content {
+    display: flex;
+    flex-direction: column;
+    flex: 1;
   }
+`
+const SuccessTitle = styled(TextDisplay1)`
+  padding: ${space.xl};
+  flex: 1;
+`
 
-  & .kirk-modal--large .kirk-successModal-footer .kirk-button {
-    margin: ${space.l} ${space.xl} 0;
-  }
+const SuccessAction = styled.div`
+  padding: ${space.xl};
+`
 
-  & .kirk-successModal-confirmButton {
-    vertical-align: bottom; /* vertical alignment to be fixed in Button */
-    border: 0; /* content alignment to be fixed in Button */
-  }
+// @todo: This should be applyed on Button component since it is part of its specification
```

Comment:
This should be part of our normalisation task (see with @grosbouddha).

Reply:
Button and ButtonLayout is a big piece of incoming work for normalization. I only work on high level layout normalization so far.
About this, If it's really a generic layout property of the button, this margin should be in the Button CSS not here in the success modal.

Reply:
This brings a lot of impact and rather create another ticket to solve the issue.. and keep it here.